app/webpacker/components/CompetitionForm/FormSections/CompDates.js

```diff
@@ -21,12 +21,26 @@ function daysUntil(date) {
 }
 
 export default function CompDates() {
```

Comment:
Maybe out of the scope of this PR, but what about automatically setting the end date when the start date is initially set? In fact, it might make sense to also automatically offset the end date whenever the start date changes, to preserve the duration.
The latter would also prevent invalid date ranges (right now the end date must be after the start date, but you can later change the start date to after the end date).

Reply:
That's a very reasonable suggestion but should be in a different PR